tools/src/main/java/com/example/core/broker/testkit/SubscribeToBuffer.java

```diff
@@ -0,0 +1,47 @@
+package com.example.core.broker.testkit;
+
+import akka.actor.ActorRef;
+
+public class SubscribeToBuffer {
```

Comment:
Style note, this is part of the `TopicBufferActor` protocol, Akka convention for Java code is to put all classes that make up the protocol of an actor as inner classes of a class called `<ActorClass>Protocol`, ie in this case `TopicBufferActorProtocol`.  Also, this class should be package private.